include/deal.II/fe/mapping.h

```diff
@@ -640,7 +640,8 @@ class Mapping : public Subscriptor
    * The function above adjusted with the variable cell_normal_vectors for the
    * case of codimension 1
    */
-  virtual void
```

Comment:
the documentation never mentions what is happening with CellSimilarity, maybe this is worth mentioning? (it seems out of date anyways with a commented out function and talking about "filling the fields of FEValues", etc.)